packages/@aws-cdk/cdk/lib/app.ts

```diff
@@ -15,6 +15,26 @@ export interface AppProps {
    */
   readonly autoRun?: boolean;
 
+  /**
+   * The output directory into which to emit synthesized artifacts.
+   *
+   * @default - If this value is _not_ set, considers the environment variable `CDK_OUTDIR`.
+   *            If `CDK_OUTDIR` is not defined, uses a temp directory.
+   */
+  readonly outdir?: string;
+
+  /**
+   * Include stack traces in construct metadata entries.
+   * @default true stack traces are included
+   */
+  readonly stackTraces?: boolean;
+
+  /**
+   * Include runtime versioning information in cloud assembly manifest
+   * @default true runtime info is included
+   */
+  readonly runtimeInfo?: boolean;
```

Comment:
There's a little consistency issue here IMO:
- `outdir` probably should be `outputDirectory` or `assemblyLocation`?
- `runtimeInfo` probably should be `runtimeInformation`, but it also introduces yet another name for version tracking, really... I would stick to ONE formulation.

Reply:
What's wrong with `outdir` and `runtimeInfo` exactly?

I've renamed the private property to `runtimeInfo` so they'll be consistent if that was the concern...